### PR TITLE
Refactor some code

### PR DESCRIPTION
**Pull request by mehdihasanpour**

### CODE REVIEW

File: tests/bootstrap.php [hunk 1]
```diff
@@ -39,16 +39,16 @@
     ],
     RouteCollector::class => [
         'is_readable' => function (string $file): bool {
-            if (stripos($file, 'non-readable.cache') !== false) {
-                return false;
-            }
-            return true;
+            return ! (stripos($file, 'non-readable.cache') !== false)
```
Comment (nbayramberdiyev) [inline tests/bootstrap.php]: You can simply do this:
```php
return stripos($file, 'non-readable.cache') === false;
```

File: tests/bootstrap.php [hunk 2]
```diff
@@ -39,16 +39,16 @@
...
+                 
+            
+             ;
         },
         'is_writable' => function (string $path): bool {
-            if (stripos($path, 'non-writable-directory') !== false) {
-                return false;
-            }
-            return true;
+            return ! (stripos($path, 'non-writable-directory') !== false)
```
Comment (nbayramberdiyev) [inline tests/bootstrap.php]: Same as above.

**Review by nbayramberdiyev — CHANGES_REQUESTED** [review]

File: tests/bootstrap.php (hunk 1 above)
Comment (mehdihasanpour) [reply tests/bootstrap.php]: Thanks, this is a better way.

Comment (nbayramberdiyev) [reply tests/bootstrap.php]: Can you also unwrap the code parentheses?

Comment (mehdihasanpour) [reply tests/bootstrap.php]: I unwrapped the code parentheses.

**Review by nbayramberdiyev — APPROVED** [review]

**Review by l0gicgate — APPROVED** [review]

**Pull request merged**